packages/examples/src/components/Example/Example.jsx

```diff
@@ -20,38 +17,50 @@ class Example extends Component {
   }
 
   onLoad = () => {
-    this.setState({
-      loaded: !this.state.loaded,
-    })
+    this.setState(state => ({
```

Comment:
Why are you using setState this way ? The other one is shorter.

Reply:
Because I rely on previous state value.
This way I am sure that I have the last state when the `setState` take place.

Functional setState is better than object IMO